espree.js

```diff
@@ -4771,7 +4771,7 @@ function parseExportNamedDeclaration() {
         do {
             isExportFromIdentifier = isExportFromIdentifier || matchKeyword("default");
             specifiers.push(parseExportSpecifier());
-        } while (match(",") && lex());
+        } while (match(",") && lex() && !match("}"));
```

Comment:
Okay let me try to find some cases that cause it to break and go from there. I don't think it will be too difficult to fix either way...